### PR TITLE
Create scosco62.json

### PR DESCRIPTION
**Pull request by scosco62**
For the purposes of the tutorial.

### CODE REVIEW

**Comment by scosco62** [comment]
Add New York City Pin.
Closes issue #293 
@githubteacher

File: _pins/scosco62.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: scosco62
+latitude: 40.7128° N
+longitude: 74.0059° W
```
Comment (jr-cologne) [inline _pins/scosco62.json]: Just remove the ° and the letters and then will it work, I think.

**Comment by cesargafer** [comment]
👍 !! 
Please feel free to work with me regarding to this training!

**Comment by griphus201** [comment]
Doing the tutorial.

**Pull request merged**